ex6/src/app/app.component.html

```diff
@@ -0,0 +1,58 @@
+<h1>Form Builder, Form Group, Form Control, Validators and ngsubmit</h1>
+
+<form [formGroup]="myForm" (ngSubmit)="onSubmit()" class="form">
+  <div class="field">
+    <label class="label" for="username">Username:</label>
+    <input type="text" 
+    id="username" 
+    class="input" 
+    formControlName="username"  
+    [ngClass]="{'invalid': myForm.get('username')?.invalid && (myForm.get('username')?.touched || myForm.get('username')?.dirty)}"/>
+    <div *ngIf="myForm.get('username')?.hasError('required') && (myForm.get('username')?.touched || myForm.get('username')?.dirty)" class="error-message">
+      Please enter a username.
+    </div>
+    <div *ngIf="myForm.get('username')?.hasError('minlength') && (myForm.get('username')?.touched || myForm.get('username')?.dirty)" class="error-message">
```

Comment:
Create a function that checks all these conditions and returns only a boolean to simplify reading the html code